test/common.py

```diff
@@ -64,9 +64,10 @@ def sslmode(self):
             mgclient.MG_SSLMODE_REQUIRE if self.use_ssl else mgclient.MG_SSLMODE_DISABLE
         )
 
-    def kill(self):
+    def terminate(self):
```

Comment:
I changed `kill` to `terminate`, because:
1. It started to cause issues when running the tests, but I am not sure why. Probably Memgraph was killed and restarted quicker than before therefore the new instance couldn't start properly.
2. I think it is better to let Memgraph terminate gracefully, e.g.: in the future we might gather log files Memgraph to see the other side of the tests also in case a failure.